### PR TITLE
Redirect widgets to another location before deleting them

### PR DESCRIPTION
**Pull request by turt2live**
This is so that shutdown hooks in the widget can correctly fire, such as Jitsi's hook to abandon its hold on the webcam.

Fixes https://github.com/vector-im/riot-web/issues/7351

### CODE REVIEW

File: src/components/views/elements/AppTile.js [hunk 1]
```diff
@@ -318,6 +318,19 @@ export default class AppTile extends React.Component {
...
+                        // its hold on the webcam. Without this, the widget holds a media
+                        // stream open, even after death. See https://github.com/vector-im/riot-web/issues/7351
+                        if (this.refs.appFrame) {
+                            // In practice we could just do `+= ''` to trick the browser
+                            // into thinking the URL changed, however I can foresee this
+                            // being optimized out by a browser. Instead, we'll just point
+                            // the iframe at a page that is reasonably safe to use in the
+                            // event the iframe doesn't wink away.
+                            // This is relative to where the Riot instance is located.
+                            this.refs.appFrame.src = '/config.json';
```
[the remark this answers is not in the document]
Comment (bwindels) [reply src/components/views/elements/AppTile.js]: @turt2live ping

[the remark this answers is not in the document]
Comment (turt2live) [reply src/components/views/elements/AppTile.js]: Is about:blank universal? Just starting the navigation appears to be enough. 

(Sorry, GitHub ate that last notification)

[the remark this answers is not in the document]
Comment (bwindels) [reply src/components/views/elements/AppTile.js]: Yeah, it' appears to be in [the html spec](https://html.spec.whatwg.org/multipag/infrastructure.html#about%3Ablank)

[the remark this answers is not in the document]
Comment (bwindels) [reply src/components/views/elements/AppTile.js]: https://stackoverflow.com/questions/4145245/does-aboutblank-work-in-all-browsers

**Pull request merged**